CONTRIBUTING.md

```diff
@@ -237,24 +237,32 @@ request. Typically reviewers will review the code for obvious errors, as well as
 test out the patch set and opine on the technical merits of the patch. Project
 maintainers take into account the peer review when determining if there is
 consensus to merge a pull request (remember that discussions may have been
-spread out over GitHub, mailing list and IRC discussions). The following
+spread out over GitHub, mailing list and IRC discussions).
+
+#### Conceptual review
+
+A review can be a conceptual review, where the reviewer leaves a comment
+ * `Concept ACK`, meaning "I agree in the general principle of this pull
+   request",
+ * or `NACK`, including a rationale why they think the change is or is not
+   worthwile. NACKs without accompanying reasoning may be disregarded.
+
```

Comment:
@sipa mentioned there is a distinction between ACKing
1) the problem intended to be solved
2) the approach being used to solve the problem

As a concept reviewer, it would be nice to communicate whether you're ACKing just #1, or also #2. This can help the author understand whether other contributors feel like this is an important problem to solve, so even those the proposed approach might not gain acceptance, it could be worthwhile for the author to go back to the drawing board to solve the problem (vs. give up entirely).

How should a concept reviewer express this? Should they state "Problem ACK, Concept NACK"?

Reply:
Can't remember the wording, but I picked "Concept ACK" and "Design ACK".

Reply:
This seems to say "Approach NACK" means "Concept ACK" which I don't think is true...

Reply:
Fixed

Reply:
I actually assumed "Approach NACK" implies "Concept ACK" somehow.
If you don't like the concept, you probably don't care too much about the approach.

So I think "Approach ACK" should imply "Concept ACK".
Not sure what "Approach NACK" should imply, but I think a NACK should be explained anyway, so that's not an issue in practice.

Reply:
Ok, reverted back